.azuredevops/deploy-core-platform-infra.yaml

```diff
@@ -51,5 +51,22 @@ extends:
                   -ResourceGroupName $(servicesResourceGroup)
                   -ResourceName $(infraResourceNamePrefix)$(nc_resource_servicebus)$(nc_instance_regionid)01                 
               - displayName: Set DNS record for $(infraResourceNamePrefix)$(nc_resource_servicebus)$(nc_instance_regionid)01
+                scriptPath: 'Set-PrivateDnsRecordSet.ps1@PipelineCommon'
+                serviceConnectionVariableName: mstAzureResourceManagerConnection
+      - name: app_configuration
```

Comment:
Do we need to create a groupedTemplate for each resource type, especially if we don't have dependencies in the deployments?

Reply:
@kbabigumira - We can add them to a single GroupedTemplate which means the deployments will run sequentially, even though there aren't any dependencies between the resources.  If we added the resources to a single main.bicep file calling out to the different module resources then we could just use a single GroupedTemplate and Bicep will handle any parallel deployments.
Because we're not going with the single main.bicep option, the only option I could see to speed the deployment up for non-dependant resources was to add them to separate GroupedTemplates.

Reply:
Resolved, we want to run some jobs concurrently, hence the need for groupedTemplates.